script.js

```diff
@@ -315,6 +323,30 @@ function displaySelectedImage(direction) {
   return null;
 }
 
+// return form errors
+function formErrors() {
+  /*  const name = document.querySelector("#name").value;
+      const message = document.querySelector("#message").value; dont need this for now */
```

Comment:
- [ ] it would be better to remove unused commented code to keep the file clean and pro

Reply:
@chawkimbarki alright then